packages/rum-core/src/common/utils.js

```diff
@@ -155,6 +155,8 @@ function checkSameOrigin(source, target) {
         isSame = checkSameOrigin(source, t)
       }
     })
+  } else if (typeof target === 'object' && target.constructor === RegExp) {
```

Comment:
you can check for `test === "function"` similar to https://github.com/elastic/apm-agent-rum-js/blob/8dda5c57a55f815f3252f7e2e769191efeefd7fe/packages/rum-core/src/performance-monitoring/transaction-service.js#L391

Reply:
Thanks, I think there are a lot of options, and each option assumes another thing.

But if we already have it, I will use it.

@vigneshshanmugam Solved